webapp/src/components/disease/index.js

```diff
@@ -4,37 +4,102 @@ import FlybaseDataGrid from 'react-flybase-datagrid';
+   {id:'evidence', name:'Evidence Code'},
+   {id:'dataProvider', name:'Association Source'},
    {id:'ref', name:'References'}
   ];
-
   return columns;
-
 }
 
-const data = [
-{ name: 'Bannayan-Riley-Ruvalcaba syndrome', assc: 'is model of', ec: 'TAS', as: 'MGI', ref: 'PMID:12345, PMID:12345'},
-{ name: 'brain glioma', assc: 'causes or contributes to condition', ec: 'TAS', as: 'FB', ref: 'PMID:12345'},
-{ name: 'Cowden disease', assc: 'is model of', ec: 'TAS', as: 'MGI', ref: 'PMID:12345'},
-{ name: 'endometrial cancer', assc: 'causes or contributes to condition', ec: 'ISS', as: 'RGD', ref: 'PMID:12345'},
-{ name: 'prostate cancer', assc: 'is model of', ec: 'IC', as: 'ZFIN', ref: 'PMID:12345'},
-{ name: 'prostate cancer', assc: 'causes or contributes to condition', ec: 'ISS', as: 'SGD', ref: 'PMID:12345'},
-{ name: 'acute lymphocytic leukemia', assc: 'is model of', ec: 'TAS', as: 'MGI', ref: 'PMID:12345'},
-{ name: 'fatty liver disease', assc: 'causes or contributes to condition', ec: 'IEP', as: 'RGD', ref: 'PMID:12345'},
-{ name: 'fatty liver disease', assc: 'is model of', ec: 'TAS', as: 'MGI', ref: 'PMID:12345'},
-{ name: 'hepatocellular carcinoma', assc: 'causes or contributes to condition', ec: 'ISS', as: 'WB', ref: 'PMID:12345'},
-{ name: 'persistent fetal circulation syndrome', assc: 'is model of', ec: 'TAS', as: 'MGI', ref: 'PMID:12345'},
-{ name: 'urinary bladder cancer', assc: 'is marker for', ec: 'ISS', as: 'RGD', ref: 'PMID:12345'},
-];
-
 class DiseaseTable extends Component {
+  constructor(props) {
+    super(props);
+
+    const diseases = this.props.data.diseases;
+
+    // var diseases = [
+    //   {
+    //     'associationType': 'contributes_to_condition',
+    //     'dataProvider': 'ZFIN',
+    //     'do_name': [
+    //       'cardiomyopathy'
+    //     ],
+    //     'evidence': [
+    //       {
+    //         'evidenceCode': 'TAS',
+    //         'pubs': [
+    //           {
+    //             'pubMedId': '27642634',
+    //             'publicationModId': 'ZFIN:ZDB-PUB-160920-5'
+    //           },
+    //           {
+    //             'pubMedId': '1233',
+    //             'publicationModId': 'ZFIN:ZDB-PUB-170406-10'
+    //           }
+    //         ]
+    //       }
+    //     ],
+    //   },
+    //   {
+    //     'associationType': 'contributes_to_condition',
+    //     'dataProvider': 'flybase',
+    //     'do_name': [
+    //       'cancer'
+    //     ],
+    //     'evidence': [
+    //       {
+    //         'evidenceCode': 'TAS',
+    //         'pubs': [
+    //           {
+    //             'pubMedId': '987654',
+    //             'publicationModId': 'ZFIN:ZDB-PUB-160920-5'
+    //           },
+    //           {
+    //             'pubMedId': '',
+    //             'publicationModId': 'ZFIN:ZDB-PUB-170406-10'
+    //           }
+    //         ]
+    //       }
+    //     ],
+    //   }
+    // ];
+
+    this.data = diseases.map((disease) => {
```

Comment:
I think it's better to add properties to `this.state` instead of directly on `this`.